modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/constants/CommerceObjectActionExecutorConstants.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.constants;
+
+/**
+ * @author Crescenzo Rega
+ */
+public class CommerceObjectActionExecutorConstants {
+
+	public static final String COMMERCE_ORDER_SYSTEM_OBJECT_DEFINITION =
+		"CommerceOrder";
+
+	public static final String KEY_SPLIT_COMMERCE_ORDER_BY_CATALOG =
+		"split-commerce-order-by-catalog";
+
+	public static final String
+		SPLIT_COMMERCE_ORDER_BY_CATALOG_CONDITION_EXPRESSION =
+			"orderStatus = 10";
+
+	public static final String
+		SPLIT_COMMERCE_ORDER_BY_CATALOG_OBJECT_ACTION_DESCRIPTION =
```

Comment:
Why having a constant? It's a label that is localized in the object framework, I would just go with a string in the class defining the action